### PR TITLE
Multiple Changes, See commits. Mainly iCal Import fixing.

### PR DESCRIPTION
**Pull request by xtech86**

### CODE REVIEW

File: plugins/jevents/jevents.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ function onContentBeforeSave($context, $data)
 			if ($result_count >= 1)
 			{
 				JFactory::getApplication()->enqueueMessage(JText::sprintf('JEV_CAT_MAN_DELETE_WITH_IDS',  $result_count), 'Warning');
-				JFactory::getApplication()->enqueueMessage(JText::_('JEV_CAT_DELETE_MSG_EVENTS_FIRST'), 'Warning');
+				JFactory::getApplication()->enqueueMessage(JText::sprintf('JEV_CAT_DELETE_MSG_EVENTS_FIRST'), 'Warning');
```
Comment (GeraintEdwards) [inline plugins/jevents/jevents.php]: You only use  JText::sprintf() when you have a variable to pass in to the format string.

File: component/site/libraries/iCalEvent.php [hunk 2]
```diff
@@ -64,6 +64,13 @@ function iCalEvent( &$db ) {
 	function store($updateNulls=false , $overwriteCreator = false) {
 		$user = JFactory::getUser();
 
+		$jinput = JFactory::getApplication()->input;
+		$curr_task = $jinput->get('task');
+		$ical_access = $jinput->get('access');
```
Comment (GeraintEdwards) [inline component/site/libraries/iCalEvent.php]: $ical_access = $jinput->getInt('access');
this ensures the data is cleaned appropriately for security reasons

File: component/site/libraries/iCalEvent.php [hunk 3]
```diff
@@ -64,6 +64,13 @@ function iCalEvent( &$db ) {
 	function store($updateNulls=false , $overwriteCreator = false) {
 		$user = JFactory::getUser();
 
+		$jinput = JFactory::getApplication()->input;
+		$curr_task = $jinput->get('task');
```
Comment (GeraintEdwards) [inline component/site/libraries/iCalEvent.php]: $jinput->getCmd('task');
this ensures the data is cleaned appropriately for security reasons

Comment (xtech86) [inline component/site/libraries/iCalEvent.php]: By default, it is cmd. So cleaned as per CMD. I will update it anyhow to make it clear.

File: component/site/libraries/iCalEvent.php (hunk 2 above)
Comment (xtech86) [inline component/site/libraries/iCalEvent.php]: Good point changed.

File: plugins/jevents/jevents.php (hunk 1 above)
Comment (xtech86) [inline plugins/jevents/jevents.php]: I made the change as I was planning on extending it to pass through a variable later, so saw no harm in adding it now.

File: component/site/views/flat/assets/css/events_css.css [hunk 4]
```diff
@@ -999,4 +999,7 @@ div#jumpto form {text-align:center; width: 93%;margin-top:10px;}
     width: 195px !important;
     text-align: left;
     display: block;
-}
\ No newline at end of file
+}
+
+/** Make sure choosen list is under input **/
+#jevents .chzn-container.chzn-container-single .chzn-results {clear:both;}
```
Comment (GeraintEdwards) [inline component/site/views/flat/assets/css/events_css.css]: In which context is this?  Are there no situations where the <select> replacement needs to be inline with the label?

Comment (xtech86) [inline component/site/views/flat/assets/css/events_css.css]: See:
https://www.dropbox.com/s/ppumj9ya6s1ecix/Screenshot%202015-09-07%2015.13.38.jpg?dl=0

Adding this CSS tells it to clear and drop to the next line and fixes the display issue.

**Pull request merged**